### DISCUSSION TITLE
Double requests

### DISCUSSION THREAD

**Discussion by burbma** [community/community#24847] [Apps, API and Webhooks]
<p>My app is getting two requests from the webhook for actions “created”, “deleted”, “added”, “removed”. Anyone else experience this? See in this photo, the two that succeeded are identical, the two that failed are identical. I don’t mean the id’s are but the contents of the requests which aren’t shown. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/552e233c15106dc3033df066ab235972ba40640e.png" data-download-href="/uploads/short-url/c9xqVTAPWV7LdXmzmykqD8E9wHI.png?dl=1" title="Screen Shot 2019-09-13 at 8.10.29 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7841889/181087058-41b4c247-a0b4-44de-88b7-cde5b0e4bf9c.png" alt="Screen Shot 2019-09-13 at 8.10.29 PM.png" data-base62-sha1="c9xqVTAPWV7LdXmzmykqD8E9wHI" width="690" height="224" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/552e233c15106dc3033df066ab235972ba40640e_2_690x224.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/552e233c15106dc3033df066ab235972ba40640e_2_1035x336.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/5/552e233c15106dc3033df066ab235972ba40640e_2_1380x448.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/552e233c15106dc3033df066ab235972ba40640e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-13 at 8.10.29 PM.png</span><span class="informations">1512×492 44.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lee-dohm** [comment, marked as answer]
<p>We haven’t seen anything like that but if you can reliably reproduce the problem, you may want to contact private support so they can take a look at the confidential data to get more information on the problem. You can contact private support at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> or via email at <a href="mailto:support@github.com">support@github.com</a>.</p>
<p>Let us know if you have more questions. </p>
